sentry/templates/configmap-snuba.yaml

```diff
@@ -25,6 +25,7 @@ data:
         "port": int({{ include "sentry.clickhouse.port" . }}),
         "user":  env("CLICKHOUSE_USER", "default"),
         "password": env("CLICKHOUSE_PASSWORD", ""),
+        "max_connections": int(os.environ.get("CLICKHOUSE_MAX_CONNECTIONS", 100)),
```

Comment:
How & when is that env variable set up?

Reply:
https://github.com/sentry-kubernetes/charts/pull/1175/commits/8a3b306e87329f57e9e1b4d9270674aad0eebffb

In this commit, we've made it possible to specify from the values.